Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt

```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Remove Intel Excess Bloom"
+path = "The Legend of Zelda: Breath of the Wild/Graphics/Intel/Remove Excess Intel Bloom"
+description = Removes excess bloom common in 23.xx and Intel older drivers. Apply before shadow fixes for no-cel. -Altros

```

Comment:
First of all, it's fine to add credits to the description (I feel like it should be more third-person-ish like a normal "Credits to Altros"). It's not a personal pack that you're giving out on Discord.

Also, the description isn't grammatically correct. I would just suggest changing the whole thing to something like:
`This workaround is for people who use the older Intel integrated GPU drivers which offer better performance but trade in graphical bugs. If you use this pack, make sure you install the 23.XX drivers or lower (see this link [insert link]). This pack fixes most issues with the older driver to look like the newer drivers.`

Maybe you can make it shorter or easier for people. I also recommend including a link to the drivers, I couldn't even find them.

Reply:
Moved credits to a comment for all rules.txt.  Modified path to /Workarounds/Intel/ for all packs. Modified description for "bloom" pack to.  description = This workaround targets users of Intel iGPU drivers 23.xx and older experiencing bloom issues. These drivers may offer better performance but have graphical glitches which. This pack aims to fix the graphical issues so the older drivers create visuals similar to the new drivers. These older drivers may be found on Intel's site https://downloadcenter.intel.com/download/27484/Graphics-Intel-Graphics-Driver-for-Windows-15-65-.